backend/kernelCI_app/utils.py

```diff
@@ -124,8 +124,12 @@ def create_filters_from_req(self, request):
             if len(request.GET.getlist(k)) > 1:
                 field = filter_term
                 values = request.GET.getlist(k)
-                for value in values:
-                    self.add_filter(field, value, "in")
+
+                if aggregate:
+                    self.add_filter(field, values, "in")
+                else:
+                    for value in values:
+                        self.add_filter(field, value, "in")
```

Comment:
In fact, I think we always want to `aggregate` here 🤔

Reply:
In the PR #426, this function were updated. So, to avoid problems, I use this parameter this particular behavior